File: [CODE]/rightoversJAVA/src/main/java/sep3/g3/rightoversjava/model/dto/UserUpdatePasswordDto.java
package sep3.g3.rightoversjava.model.dto;

public class UserUpdatePasswordDto {
    private String username;
    private String oldPassword;
    private String newPassword;

    public UserUpdatePasswordDto(String username, String oldPassword, String newPassword) {
        this.username = username;
        this.oldPassword = oldPassword;
        this.newPassword = newPassword;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getOldPassword() {
        return oldPassword;
    }

    public void setOldPassword(String oldPassword) {
        this.oldPassword = oldPassword;
    }

    public String getNewPassword() {
        return newPassword;
    }

    public void setNewPassword(String newPassword) {
        this.newPassword = newPassword;
    }
}

File: [CODE]/rightoversBlazorNWEB/Domain/DTOs/UserUpdatePasswordDto.cs
namespace Domain.DTOs;

public class UserUpdatePasswordDto
{
    public string Username { get; }
    public string OldPassword { get; }
    public string NewPassword { get; }

    public UserUpdatePasswordDto(string username, string oldPassword, string newPassword)
    {
        Username = username;
        OldPassword = oldPassword;
        NewPassword = newPassword;
    }
}